Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -1,104 +0,0 @@
-.. only:: not (epub or latex or html)
```

Comment:
I think we have some leftovers to `k8s-install-gke` and friends, e.g. in `install/kubernetes/cilium/README.*` and `.github/ISSUE_TEMPLATE/release_template.md`. I suppose we should update them to the new guides as well.